test/Interop/Cxx/foreign-reference/Inputs/cxx-functions-and-methods-returning-frt.h

```diff
@@ -1,3 +1,4 @@
+#include <functional>
```

Comment:
Minor: I think the convention is that `#pragma once` should go above any includes.

Reply:
This still needs to be resolved